webpack.config.js

```diff
@@ -25,7 +25,9 @@ module.exports = {
   output: {
     path: __dirname + '/dist',
     publicPath: '/',
-    filename: 'bundle.js'
+    filename: 'bundle.js',
+    devtoolModuleFilenameTemplate        : '[absolute-resource-path]',
+    devtoolFallbackModuleFilenameTemplate: '[absolute-resource-path]?[hash]'
```

Comment:
[fixing vs code not able to find source map](https://github.com/aurelia/skeleton-navigation/issues/758)